backends/dpdk/elimTypedef.cpp

```diff
@@ -0,0 +1,15 @@
+#include "elimTypedef.h"
```

Comment:
This should probably be done in the midend; we already have such a pass.

Reply:
fixed.